src/main/java/io/specto/hoverfly/junit/api/view/ResponseDiffForRequestView.java

```diff
@@ -0,0 +1,28 @@
+package io.specto.hoverfly.junit.api.view;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ResponseDiffForRequestView {
+    private final SimpleRequestDefinitionView request;
+    private final List<String> diffMessages;
```

Comment:
You may want to update it now as the JSON schema has changed.

Reply:
Sure, I'll do that this week. I have already started working on it.

Reply:
updated in 57bc172